sql/flights.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS flights;
+
+CREATE TABLE flights (
+    id		INT DEFAULT 1,
+    link	VARCHAR(250),
```

Comment:
This should probably be `NOT NULL`; what would it mean to have a `NULL` link? Also links can be quite a bit longer than this, so it would probably be better to store this as a `TEXT` field instead.